index.js

```diff
@@ -1,4 +1,5 @@
 const jsep = require('jsep');
+const _ = require('lodash');
```

Comment:
Lodash would increase the size of this library quite a bit — I'd prefer not to add any new dependencies. With the API change suggested below, I don't think this PR should need it anymore.